src/main/java/org/gradlex/buildparameters/BuildParameterGroup.java

```diff
@@ -26,21 +26,44 @@
 
 public abstract class BuildParameterGroup {
 
+    private static final Action<Object> NO_OP = o -> {};
+
     final Identifier id;
 
     @Inject
     public BuildParameterGroup(Identifier identifier) {
         this.id = identifier;
     }
 
+    /**
+     * @since 1.2
+     */
```

Comment:
We haven't discussed how we want to deal with JavaDocs on the plugin DSL types. I personally find the since tag very helpful when figuring reading code. That's why I added it here. But I'm also fine with dropping it.